ios/edge/AppDelegate.m

```diff
@@ -1,5 +1,5 @@
 #import "AppDelegate.h"
-#import "Edge-Swift.h"
+#import "EdgeApiKey-Swift.h"
```

Comment:
This last one seems wrong to me. The name Edge-Swift comes from the Xcode project settings, and Xcode itself auto-generates this file.